### PR TITLE
docs: error messages

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

File: docs/docs/manuals/error-messages.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Error Messages
```
Comment (fforootd) [inline docs/docs/manuals/error-messages.md:2]:
```suggestion
title: Troubleshoot
```

File: docs/docs/manuals/error-messages.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+You will find some possible error messages here, what the problem is and what some possible solutions can be.
+
+## User Agent does not correspond
+
+This error appeared for some users as soon as they were redirected to the login page of ZITADEL.
+
+We only found this issue with iPhone users and it was dependent on the settings of the device.
+
+### Solution
```
Comment (fforootd) [inline docs/docs/manuals/error-messages.md:13]: I would recommend that we also provide a problem statement. E.g cookies are blocked which affects ZITADELs function

File: docs/docs/manuals/error-messages.md (hunk 1 above)
Comment (fforootd) [reply docs/docs/manuals/error-messages.md:2]: I think we should call this troubleshoot, as it is the location for users to visit when something brakes.

File: docs/docs/manuals/error-messages.md (hunk 2 above)
Comment (fgerschwiler) [reply docs/docs/manuals/error-messages.md:14]: i think this makes sense, I also added why we need the cookies

**Review by fforootd — DISMISSED** [review]

**Review by fforootd — APPROVED** [review]

**Pull request merged**